README.md

```diff
@@ -1,131 +1,102 @@
-# Github Contribution Dashboard
+# 18F Projects Dashboard
 
-[![Build Status](https://travis-ci.org/chillu/github-dashing.png?branch=master)](https://travis-ci.org/chillu/github-dashing)
+[![Build Status](https://travis-ci.org/18F/github-dashing.png?branch=master)](https://travis-ci.org/18F/github-dashing)
 
-Dashboard to monitor the health of github projects based on their contribution statistics.
+Dashboard to monitor the health of 18F GitHub repos.
 
- - Aggregates usage data across multiple repos from the Github API
- - Widgets support aggregate statistics of multiple repos or even all repos within an organization.
- - Contributor scores based on activity in commits as well as in comments and pull requests.
- - [Travis CI](http://travis-ci.org) build status across multiple branches
- - Trend projections for current month on issues opened, issues closed and pull requests
- - Quick integration of other data sources through a common widget framework
- - Easy hosting through [Heroku](http://heroku.com)
+Current widgets include the following:
 
-All visualizations are optimized to encourage direct action by individuals, so prefers short-term trends and
-relative measures over long-term data. For example, the leaderboard only inspects the last 30 days
-of contributions, allowing new contributors to get to the top more easily.
+Available at https://project-dashboard.18f.gov/default:
```

Comment:
I'll remove line 7 and add make the sentence on lines 9 and 18 start with `Widgets available`.